.github/workflows/build.yml

```diff
@@ -29,6 +24,15 @@ jobs:
       - name: Run lint
         run: |
           make lint
+      - name: Install and start Redis
+        run: |
+          make testdeps
+          make start-redis
```

Comment:
Do we need to start redis here? I think this can be handled by the test suite with startRedisServer. It's probably better to do it in code, we may have more control over the configuration.

Reply:
It's because I prioritized making testing easier in a local environment.
Some people might run Redis locally, but others might be in a Docker container.

Reply:
Ummm, but I also don't want the test to fail because of environmental differences.
Thinking so, using `startRedisServer` isn't a bad idea... :thinking:

Reply:
Yeah, I was also thinking what if the redis-server instance already running on a user's machine have some conflicting configs like cluster-enabled for example, it will break normal tests

Reply:
OK, I'll change the tests later to use `startRedisServer` to start the testing Redis.